### ISSUE TITLE
Close button dissapears

### ISSUE THREAD

**Issue by leviti-gasandov** [UgnisSoftware/react-spreadsheet-import#182]
I dont know if there is a bug, but when the spreadsheet is open directly the close button appears and works correctly. But when I sent the onClose functionality into a button, the cose button of the spreadsheet dissapears. I attach the code and some screenshots for references.
<img width="1430" alt="Screen Shot 2023-06-28 at 11 33 10" src="https://github.com/UgnisSoftware/react-spreadsheet-import/assets/137426362/182a984e-bd9a-43ca-8f30-a1fd9667255c">
<img width="1420" alt="Screen Shot 2023-06-28 at 11 33 48" src="https://github.com/UgnisSoftware/react-spreadsheet-import/assets/137426362/bd700ff7-5fa9-4969-af09-6617317e1fc6">
<img width="1422" alt="Screen Shot 2023-06-28 at 11 33 59" src="https://github.com/UgnisSoftware/react-spreadsheet-import/assets/137426362/9b4d5ea6-7be1-4428-8d64-1754dc408c39">

```

const [isOpen, setIsOpen] = useState(true);
  const onClose = () => {
    setIsOpen((prev) => !prev);
  };
 const onSubmitGuestList = (data: any) => {
    console.log(data);
  };
return (
 <Toolbar>
            <ReactSpreadsheetImport
              isOpen={isOpen}
              onClose={onClose}
              onSubmit={onSubmitGuestList}
              fields={guestListFormCVSValidator}
              customTheme={{
                components: {
                  Button: {
                    defaultProps: {
                      colorScheme: "orange",
                    },
                  },
                },
              }}
            />
            <Button variant="outlined" color="secondary" onClick={onClose}>
              Upload CSV
            </Button>
          </Toolbar>
          )
```

**Comment by lokacoding** [comment]
I have the same issue.

No Close button is shown!

 <ReactSpreadsheetImport
          isOpen={true}
          onClose={onClose}
          onSubmit={onSubmit}
          fields={fields}
          customTheme={{
            components: {
              Button: {
                baseStyle: {
                  borderRadius: "8px",
                },
                defaultProps: {
                  colorScheme: "purple",
                },
              },
            },
          }}
          translations={{...}}
        />

**Comment by JulitorK** [comment]
Hi! Thanks for reporting. 
I tested with your snippet and didn't manage to recreate it. I don't have much to go on with without a sandbox. 
Do you by any chance use chakra for the rest of your project? Any custom styles applied to modals?

What comes up if you search for below string in devtools html? Is it there? What positioning is applied?: 
`M6.2253 4.81108C5.83477 4.42056 5.20161 4.42056 4.81108 4.81108C4.42056 5.20161 4.42056 5.83477 4.81108 6.2253L10.5858 12L4.81114 17.7747C4.42062 18.1652 4.42062 18.7984 4.81114 19.1889C5.20167 19.5794 5.83483 19.5794 6.22535 19.1889L12 13.4142L17.7747 19.1889C18.1652 19.5794 18.7984 19.5794 19.1889 19.1889C19.5794 18.7984 19.5794 18.1652 19.1889 17.7747L13.4142 12L19.189 6.2253C19.5795 5.83477 19.5795 5.20161 19.189 4.81108C18.7985 4.42056 18.1653 4.42056 17.7748 4.81108L12 10.5858L6.2253 4.81108Z`

**Comment by JulitorK** [comment]
The original `right` and `top` properties should be as follows:
```
    right: 14px;
    top: 20px;
```
Have added any custom styles to it?

**Comment by JulitorK** [comment]
#196 provides a way to fix it, released with 4.1.2.  
If source of overriding can't be found, add this to your custom theme: 
`customTheme={{ components: { Modal: { baseStyle: { closeModalButton: { top: "14px", right: "20px" } } } } }}`

Feel free to comment here for me to reopen if you have further issues.

**Comment by lokacoding** [comment]
I forgot to mention that I'm not using latest version, because of some other dependencies my project is still on **React 17** and use **version 2.0.8**.
If i add this manually to my customTheme it still not works.

**Comment by JulitorK** [comment]
@lokacoding, I have backported these changes and released with v3.1.1. This version still uses React 17, so it should work for you.
